lib/specinfra/backend/docker.rb

```diff
@@ -79,6 +79,9 @@ def create_and_start_container
 
         @container = ::Docker::Container.create(opts)
         @container.start
+        while @container.json['State'].key?('Health') && @container.json['State']['Health']['Status'] == "starting" do
+          sleep 0.5
```

Comment:
.5s seems to be a fair amount of time to wait without looping too quickly. We could reduce that if necessary.